### PR TITLE
Exercise Week 3 done

### PR DESCRIPTION
**Pull request by Lealdos**

### CODE REVIEW

File: student-cvs/Leal2-6465/styles.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    background-color: rgba(118, 130, 130, 0.895);
+    margin: 0 auto;
+    max-width: 30em;
+    line-height: 1.4;
+    padding: 4em 1em;
+    color: #231f1f;
+    font-family: 'Comfortaa', cursive
+  }
+
+  h1,  h3 {
```
Comment (Josofico) [inline student-cvs/Leal2-6465/styles.css:14]: Please, fix the tab/space

**Review by Josofico — APPROVED** [review]
Done!

**Review by Josofico — APPROVED** [review]
Well done

File: student-blogs/Leal2-6465/styles.css [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+  text-decoration: none;
+}
```
Comment (Pancratzia) [inline student-blogs/Leal2-6465/styles.css:90]: There might be a way to summarize this. However, it's just a suggestion

**Review by Pancratzia — APPROVED** [review]
Great job! I really liked the color change in the background of your CV when switching from mobile to desktop. Additionally, you implemented the form and blog entries correctly

**Comment by Lealdos** [comment]
# Css reset modified 

## I took your advice and modified my CSS reset is now easier to read and understand

### If you can help me and check again my PR  that will be great
 
@Pancratzia 
@Josofico

**Review by Pancratzia — APPROVED** [review]
Well done!

**Review by Josofico — APPROVED** [review]

**Pull request merged**